rust/src/delta.rs

```diff
@@ -1153,6 +1189,17 @@ impl<'a> DeltaTransaction<'a> {
     }
 }
 
+fn log_entry_from_actions(actions: &[Action]) -> Result<String, serde_json::Error> {
+    let mut jsons = Vec::<String>::new();
```

Comment:
you can actually write it in a more efficient way by writing serialized action json directly into a string buffer, that way we can avoid the vector join at the end.